app/models/product.rb

```diff
@@ -0,0 +1,17 @@
+class Product < ApplicationRecord
```

Comment:
how to know whether it's a drink of food
`Food items can be food / drink / combo`

Reply:
Added a column for that named, kind.
and added scopes for them too.